src/main/java/com/twistercambodia/karasbackend/customer/entity/Customer.java

```diff
@@ -20,7 +21,15 @@ public class Customer {
 
     @OneToMany
     @JoinColumn(name = "customer_id")
-    private Set<Vehicle> vehicles;
```

Comment:
Let's use Set instead of List, because List allows for duplicate entries. (Let's say I add vehicle with Id 1 and Id 1, it will create two vehicle with Id 1)